src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProcessor.cs

```diff
@@ -19,8 +20,10 @@ internal class ConsoleLoggerProcessor : IDisposable
         public IConsole Console;
         public IConsole ErrorConsole;
```

Comment:
```suggestion
        public IConsole Console { get; }
        public IConsole ErrorConsole { get; }
```

These don't need to be settable anymore.